app/packs/src/components/nmriumWrapper/NMRiumDisplayer.js

```diff
@@ -100,7 +100,13 @@ export default class NMRiumDisplayer extends React.Component {
         const nmrWrapperActionType = eventData.data.actionType;
         if (nmrWrapperActionType !== '') {
           const nmriumData = eventData.data;
-          this.setState({ nmriumData });
+          const { version } = nmriumData;
+          if (version > 3) {
+            this.setState({ nmriumData: nmriumData.data });
+          }
+          else {
+            this.setState({ nmriumData });
+          }
```

Comment:
Can we improve this to act if things go wrong (--> error message would suffice)?
If data version 5, for example, changes the format again, we do not handle it. Users will not get feedback. :/